src/main/resources/data/minecolonies/researches/civilian/1_firstaid.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "firstaid",
+  "name": "First Aid",
```

Comment:
Can we also add a description here? the autogenerated descriptions aren't great, and aren't localizable.

Reply:
Yeah, I've got some tools to bulk-add json traits.  In a perfect world, I was hoping to get "name" to be that human-facing one, but I think that transition is going to involve a lot of pecking at ui-side stuff I don't understand yet.

 Always good to be forward-thinking toward that point, though.  Want a separate translation "key" and "description" field?  Or an object containing them?

Reply:
Simply put, id==code/json reference, name==displayname (and should be capable of being a tranlation key) and description is what it sounds like, again capable of being a translation key (and ours should be)

Reply:
Specifically, I'd like to see Description be a better write up of what the effect actually does. Gilded Hammer is a good example where the name is only partially indicative of the result of the unlock ability. There are some, like endknowlege that have potentially multiple results of a single unlock, and it would be great for the description to give us a mechanism for communicating that.

Reply:
if we're going to localize it anywaym I don't think there's a need for both name and id fields, I'd much rather form the translation key dynamically from the id.

As for the descriptions, we probably just need to have an autogenerated system with localization. It won't be hard to convert something from this effects list to something quite descriptive with localization. Maybe design the system with a fallback for the more complex use cases. If you like, you can leave this to me when I update the UI and pick on me then if it doesn't work.

Reply:
Yeah we should get the name from the id, and query that from the GUI, no need to have that anywhere in the datapacks

Reply:
(same for effects, same for descriptions)